lib/validateService.js

```diff
@@ -34,7 +38,7 @@ export async function validateService(publicServiceId){
     formStore.parse(form.form.meta, FORM_GRAPHS.metaGraph, 'text/turtle');
     formStore.parse(form.form.source, FORM_GRAPHS.sourceGraph, 'text/turtle');
 
-    const submittedResource = new rdflib.NamedNode("http://data.lblod.info/id/public-service/"+publicServiceId);
```

Comment:
Don't rely on the structure of an uri to get data. Stuff can come from everywhere (see message commit)